docs/extending.txt

```diff
@@ -549,3 +549,44 @@ arguments.  Example use:
         endswith=endswith) + t
 
     site.template_hooks['page_header'].append(greeting, True, u'Nikola Tesla', endswith=u'!')
+
+Shortcodes
+==========
+
+Some (hopefully all) markup compilers support shortcodes in these forms::
```

Comment:
And I can add that one when I add a bunch of builtin template-based shortcodes in the next branch. GOOD.